packages/browser-extension/source/localization/localizationInterfaces.ts

```diff
@@ -0,0 +1,10 @@
+export interface LocaleWithPosition {
+    text: string;
+    position: number;
+}
+
+export interface Position {
+    text: string;
```

Comment:
please use better naming (this is too general naming)

check formatjs/react-intl documentation and inspire by bettern naming (see the <FormattedMessage attributes and use theme here)